### DISCUSSION TITLE
Wiping sensitive data

### DISCUSSION THREAD

**Discussion by hakavlad** [samuel-lucas6/Kryptor#70] [Q&A] [answered]
>Sensitive data, such as cryptographic keys, should be erased from memory after use, to reduce the risk that a bug will expose it to the outside world. 

-- https://www.gnu.org/software/libc/manual/html_node/Erasing-Sensitive-Data.html

>Privacy
>- Sensitive data is wiped from the heap asap

-- https://github.com/cryptomator/cryptomator

Please clarify if Kryptor does the same.

>Sensitive bytes SHOULD be zeroed from memory once they are no longer required.

-- https://www.kryptor.co.uk/specification

Does Kryptor actually do this? This is not described in Features and README.

**Comment by samuel-lucas6** [comment, marked as answer]
Yes, Kryptor attempts to zero sensitive bytes. However, it isn't guaranteed to work in this project or in certain programming languages, which is why that's a **SHOULD** rather than a **MUST**.
